userspace/libgolf2/aes_ecb_syscalls.h

```diff
@@ -0,0 +1,114 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
```

Comment:
I just realized this file's name is "aes_ecb_syscalls.h", but it contains ECB + CTR + CBC APIs.

Should it be renamed to aes_syscalls.h?

Reply:
The standard Tock syscall interface (not totally working) doesn't support ECB mode. I was worried about a name collision.

Reply:
How about aes_h1b_syscalls.h? My concern with aes_ecb_syscalls.h is that aes_ecb_syscalls indicates it *only* contains ECB syscalls, which is inaccurate.

Reply:
I think I'd prefer aes_syscalls_h1b.h; does that work for you? I can add comments about how it is different than the standard syscall header file.